units/host_monitoring_external_termination.go

```diff
@@ -134,16 +133,18 @@ func handleExternallyTerminatedHost(ctx context.Context, id string, env evergree
 		if cloudStatus != cloud.StatusTerminated && (h.UserHost || h.StartedBy != evergreen.User) {
 			return false, errors.New("non-agent host is not already terminated and should not be terminated")
 		}
+
 		if h.SpawnOptions.SpawnedByTask {
```

Comment:
It seems like checking if it's spawned by a task is redundant if it's already checked in the function.